File origsrc/log4shib-2.0.1/Portfile is a regular file while file src/log4shib-2.0.1/Portfile is a symbolic link
--- origsrc/log4shib-2.0.1/src/Makefile.am	2018-07-07 03:39:40.000000000 +0900
+++ src/log4shib-2.0.1/src/Makefile.am	2021-11-02 13:13:34.596428800 +0900
@@ -48,5 +48,5 @@ liblog4shib_la_SOURCES = \
 	PortabilityImpl.cpp \
 	AbortAppender.cpp
 
-liblog4shib_la_LDFLAGS = -version-info @LT_VERSION@
+liblog4shib_la_LDFLAGS = -no-undefined -version-info @LT_VERSION@
 liblog4shib_la_LIBADD = $(PTHREAD_LIBS)
